client/src/app/page/navbar/navbar.component.ts

```diff
@@ -27,13 +27,12 @@ export class NavbarComponent implements OnInit {
     this.navItemsService.getNavList()
       .subscribe(list => this.menuList = list);
     this.userType = this.userService.getUserType();
-    this.active = false;
     this.avatar = 'assets/img/navbar-symbol-desk.png';
   }
 
-  currentPage(): boolean {
-    this.menuList.map(item => item.current = item.id === 'upcoming-tasks');
-    console.log(this.menuList);
+  currentPage(currentRouter): boolean {
+    this.menuList.map(item => item.current = item.router === currentRouter);
```

Comment:
why not .find?

Reply:
he returns an element that satisfies the condition, and we need to move the entire array and change the value to all